TrackCat/settings.py

```diff
@@ -100,3 +108,15 @@
 	from .settings_local import *
 except ImportError as e:
 	pass
+
+AUTH_PROFILE_MODULE = 'api.UserProfile'
+LOGIN_URL = '/login/'
+LOGIN_REDIRECT_URL = '/'
+LOGIN_ERROR_URL = '/login-error/'
+SOCIAL_AUTH_NEW_USER_REDIRECT_URL = '/'
+SOCIAL_AUTH_ENABLED_BACKENDS = ('github')
+SOCIAL_AUTH_DEFAULT_USERNAME = 'new_social_auth_user'
+SOCIAL_AUTH_GITHUB_KEY = ''
+SOCIAL_AUTH_GITHUB_SECRET = ''
+
+from settings_local import *
```

Comment:
You already have this import a [few lines above](https://github.com/anuschka/TrackCat/blob/issue-118-social-auth/TrackCat/settings.py#L107-L110) - it's best to keep that as the last part of the file and move the AUTH, LOGIN and SOCIAL stuff above with other authentication settings. See the [codeweekeu settings file](https://github.com/codeeu/coding-events/blob/master/codeweekeu/settings.py) for some organization tips and the way you can add comments, so we'll know what belongs where.